tools/buildtools/ambertools/parmconv.xml

```diff
@@ -197,6 +198,10 @@ for system in systems:
       <param name="stripmask_receptor" value=":NA,CL,SOL,WAT"/>
       <!-- test a fairly complex selection. All backbone oxygens in residues 1-500 but not in water, lysine or arginine -->
       <param name="stripmask_solvatedcomplex" value=":1-500@O&amp;!(:WAT|:LYS,ARG)"/>
+      <assert_stdout>
+        <has_text text="Getting coordinates"/>
+        <has_text text="Using topology directory"/>
```

Comment:
just to mention there is also a `<not_has_text text="chr8" />`  ... maybe that is also useful to catch warnings/errors ...

Reply:
Good idea, I stuck with the 'good' messages as not sure of all possible errors.